RetinaNet.py

```diff
@@ -24,16 +24,20 @@ def __load_data(self, dataset, num_images=200):
 
         # TODO
 
-        return ...
+        return None
     
     """
     Helper method for training
     """
-    def __resize_image(self, height=200, width=200):
-
-        # TODO
-
-        return ...
+    def __resize_image(self, size=200):
```

Comment:
keep height and width as two separate params as we may need the freedom to resize to a rectangle. Also, it seems that I forget to include the param for the image... where does im_pth come from in the code?

Reply:
im_pth will be the missing param. I'll add it.